tools/data_gen/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "gen_json"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+calyx = { path = "/Users/calebkim/learning/calyx/calyx"}
```

Comment:
This is not quite right. You should instead use a relative path here `../../calyx`.